modules/ossm-configuring-jaeger.adoc

```diff
@@ -31,101 +30,148 @@ The default Jaeger parameters specified in the `ServiceMeshControlPlane` are as
 ====
 
-Elasticsearch is a memory intensive application. The initial set of nodes created by the {product-title} installation may not be large enough to support the Elasticsearch cluster. Additional nodes must be added to the  cluster if you want to run with the recommended amount (or more) memory. Each Elasticsearch node can operate with a lower memory setting though this is not recommended for production deployments.
 
-You should modify the default Elasticsearch configuration to match your use case. You can adjust both the CPU and memory limits for each component by modifying the resources block with valid memory and CPU values.
+[id="ossm-configuring-jaeger-elasticsearch_{context}"]
+== Configuring Elasticsearch
+
+The default Jaeger deployment strategy uses the `all-in-one` template so that the installation can be completed using minimal resources.  However, because the `all-in-one` template uses in-memory storage, it is only recommended for development, demo, or testing purposes and should NOT be used for production environments.
 
+If you are deploying {ProductShortName} and Jaeger in a production environment you must change the template to the `production` template, which uses Elasticsearch for Jaeger's storage needs.
 
-.Default Jaeger parameters for Elasticsearch in production
+Elasticsearch is a memory intensive application. The initial set of nodes specified in the default {product-title} installation may not be large enough to support the Elasticsearch cluster.  You should modify the default Elasticsearch configuration to match your use case and the resources you have requested for your {product-title} installation. You can adjust both the CPU and memory limits for each component by modifying the resources block with valid CPU and memory values. Additional nodes must be added to the  cluster if you want to run with the recommended amount (or more) of memory. Ensure that you do not exceed the resources requested for your {product-title} installation.
+
+.Default "production" Jaeger parameters with Elasticsearch
 [source,yaml]
 ----
   apiVersion: maistra.io/v1
   kind: ServiceMeshControlPlane
   spec:
     istio:
       tracing:
-        jaeger:
-          template: production-elasticsearch
-          elasticsearch:
-            nodeCount: 3
-            redundancyPolicy:
-            resources:
-              requests:
-                memory: "16Gi"
-                cpu: "1"
-              limits:
-                memory: "16Gi"
+      enabled: true
+      ingress:
+        enabled: true
+      jaeger:
+        template: production-elasticsearch
+        elasticsearch:
+          nodeCount: 3
+          redundancyPolicy:
```

Comment:
nit: remove this or document possible values.

Reply:
The values are documented here https://docs.openshift.com/container-platform/4.1/logging/efk-logging-deploying-about.html

Reply:
I was considering linking to that file, I'll do that in my revision.